server/service/integration_mdm_test.go

```diff
@@ -2,8 +2,7 @@ package service
 
 import (
 	"bytes"
-	"context"
-	"crypto/md5" // nolint:gosec // used only for tests
+	"context" // nolint:gosec // used only for tests
```

Comment:
The `nolint` was for `crypto/md5`, it is not relevant for `context`.
```suggestion
	"context"
```

Reply:
great catch, thank you!